internal/blockdb/sql_test.go

```diff
@@ -26,12 +29,9 @@ func migratedDB() *sql.DB {
 }
 
 func TestConnectDB(t *testing.T) {
-	f, err := os.CreateTemp("", t.Name())
-	require.NoError(t, err)
-	defer f.Close()
-	defer os.RemoveAll(f.Name())
-
-	db, err := ConnectDB(context.Background(), f.Name(), 10)
+	file := filepath.Join(os.TempDir(), strconv.FormatInt(time.Now().UnixMilli(), 10), "test", t.Name()+".db")
+	defer os.RemoveAll(file)
```

Comment:
Not using t.TempDir() here, combined with only the call to `os.RemoveAll(file)`, will leave the `$TMPDIR/$(now)/test` directory sitting around after test. `t.TempDir()` makes more sense than `os.TempDir()` here.

Reply:
Oh I see: `t.TempDir()`. TIL; that's a great feature! I was getting confused with `os.TempDir`. A couple characters make all the difference.